data/developerhub/applications.json

```diff
@@ -168,7 +168,19 @@
       "complexity": ["beginner"],
```

Comment:
I also realized that we have a dangling "beginner" in the `complexity` section from the sqs-fargate sample now, i think it should have been "basic". Could you please correct this as well in this PR?